dlt/destinations/impl/qdrant/qdrant_client.py

```diff
@@ -357,18 +387,28 @@ def get_stored_schema(self) -> Optional[StorageSchemaInfo]:
         """Retrieves newest schema from destination storage"""
         try:
             scroll_table_name = self._make_qualified_collection_name(self.schema.version_table_name)
+            p_schema_name = self.schema.naming.normalize_identifier("schema_name")
+            # this works only because we create points that have no vectors
+            # with decreasing ids. so newest (lowest ids) go first
+            # we do not use order_by because it requires and index to be created
+            # and this behavior is different for local and cloud qdrant
+            # p_inserted_at = self.schema.naming.normalize_identifier("inserted_at")
             response = self.db_client.scroll(
                 scroll_table_name,
                 with_payload=True,
                 scroll_filter=models.Filter(
                     must=[
                         models.FieldCondition(
-                            key="schema_name",
+                            key=p_schema_name,
                             match=models.MatchValue(value=self.schema.name),
                         )
                     ]
                 ),
                 limit=1,
+                # order_by=models.OrderBy(
```

Comment:
how do the tests pass with this commented out? this does not look right to me, we should always get the newest schema, no?

Reply:
Ah ok, it's even in the pr comment, so you know.. ;)

Reply:
let's talk about it and make qdrant work again